x/lazyledgerapp/types/tx.pb.go

```diff
@@ -547,8 +547,6 @@ func (c *msgClient) PayForMessage(ctx context.Context, in *MsgWirePayForMessage,
 type MsgServer interface {
 	// PayForMessage allows the user to post data to made be available.
 	PayForMessage(context.Context, *MsgWirePayForMessage) (*MsgPayForMessageResponse, error)
-	// PayForMessage allows the user to post data to made be available.
-	SignedTransactionDataPayForMessage(context.Context, *SignedTransactionDataPayForMessage) (*SignedTransactionDataPayForMessageResponse, error)
```

Comment:
Why are there changes in this file with no changes in the corresponding proto file?

Reply:
I had manually changed them in the last PR when I added the handler for `SignedTransactionDataPayForMessage` :grimacing: , which was definitely the wrong approach to change `MsgServer` without adding an rpc endpoint. It was also my fault for force pushing the repo, otherwise someone would have probably caught it and informed me that it was a bad idea.

Reply:
does this need to be a separate PR?

Reply:
No worries! Thanks for the explanation. Hmm, yeah probably a good idea to rip out the rpc/endpoint/handler changes and clean that up in a separate PR.

Reply:
Actually, if that means self-contained small changes, then we can actually address it in this PR. Probably good to open a tiny issue explaining the changes for reference though (amend a `closes #XXX` referencing that issue too in the opening comment).